docs/reference/koin-mp/kmp.md

```diff
@@ -141,4 +141,4 @@ struct ContentView: View {
 
 ### New Native Memory Management
 
-Activate experiemental with root [gradle.properties](http://gradle.properties) properrties:
\ No newline at end of file
+Activate experiemental with root [gradle.properties](http://gradle.properties) properrties:
```

Comment:
still "properrties" :P 

can you fix it?

Reply:
Done